### PR TITLE
CASMCMS-9045 - IMS clarify configuring remote build nodes.

### PR DESCRIPTION
**Pull request by dlaine-hpe**
# Description

https://jira-pro.it.hpe.com:8443/browse/CASMCMS-9045

Clarify the documentation for the system admins on how to customize remote build nodes on their particular system.

# Checklist
- [X] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [X] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

### CODE REVIEW

**Review by jsl-hpe — APPROVED** [review]

File: operations/image_management/Configure_a_Remote_Build_Node.md [hunk 1]
```diff
@@ -61,6 +61,29 @@ There are only two requirements for using a compute node as a remote build node:
 * Have Podman installed and configured
 * Allow IMS access via SSH key
 
+Each system is unique and the node chosen to run remote jobs needs to be manually configured based on the
```
Comment (jsollom-hpe) [inline operations/image_management/Configure_a_Remote_Build_Node.md]: , and

File: operations/image_management/Configure_a_Remote_Build_Node.md [hunk 2]
```diff
@@ -61,6 +61,29 @@ There are only two requirements for using a compute node as a remote build node:
...
+requirements and capabilities of the system and that particular node. It is not possible to list all the
+configuration steps to optimize a particular node, the directions below are only intended to make a
```
Comment (jsollom-hpe) [inline operations/image_management/Configure_a_Remote_Build_Node.md]: , --> ;

File: operations/image_management/Configure_a_Remote_Build_Node.md [hunk 3]
```diff
@@ -61,6 +61,29 @@ There are only two requirements for using a compute node as a remote build node:
...
+
+Some things to consider are:
+
+1. Volume mounts.
+
+    There needs to be adequate space on the node to handle the images that are being created or customized
+    on the remote node. The faster the volume access, the faster the image customization. Ways to improve
+    performance beyond the mounts described below can include:
+
+    * If there are local disk drives on the node, use them.
```
Comment (jsollom-hpe) [inline operations/image_management/Configure_a_Remote_Build_Node.md:78]: :D

File: operations/image_management/Configure_a_Remote_Build_Node.md (hunk 1 above)
Comment (dlaine-hpe) [reply operations/image_management/Configure_a_Remote_Build_Node.md]: Disagree - replaced 'and' with 'so'.

File: operations/image_management/Configure_a_Remote_Build_Node.md (hunk 2 above)
Comment (dlaine-hpe) [reply operations/image_management/Configure_a_Remote_Build_Node.md]: Done

**Review by jsollom-hpe — APPROVED** [review]

File: operations/image_management/Configure_a_Remote_Build_Node.md [hunk 4]
```diff
@@ -61,6 +61,29 @@ There are only two requirements for using a compute node as a remote build node:
 * Have Podman installed and configured
 * Allow IMS access via SSH key
 
+Each system is unique so the node chosen to run remote jobs needs to be manually configured based on the
+requirements and capabilities of the system and that particular node. It is not possible to list all the
+configuration steps to optimize a particular node. The directions below are only intended to make a
```
Comment (nrockershousen) [inline operations/image_management/Configure_a_Remote_Build_Node.md]:
```suggestion
configuration steps to optimize a particular node. The following directions are only intended to make a
```

**Review by nrockershousen — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**